gravitee-am-repository/gravitee-am-repository-jdbc/src/main/java/io/gravitee/am/repository/jdbc/oauth2/api/JdbcRefreshTokenRepository.java

```diff
@@ -89,21 +90,21 @@ public Completable deleteByUserId(String userId) {
     }
 
     @Override
-    public Completable deleteByDomainIdClientIdAndUserId(String domainId, String clientId, String userId) {
+    public Completable deleteByDomainIdClientIdAndUserId(String domainId, String clientId, UserId userId) {
         LOGGER.debug("deleteByDomainIdClientIdAndUserId({},{},{})", domainId, clientId, userId);
         return monoToCompletable(getTemplate().delete(JdbcRefreshToken.class)
-                .matching(Query.query(where(SUBJECT).is(userId)
+                .matching(Query.query(where(SUBJECT).is(userId.getInternalSubject())
```

Comment:
**issue**: internalSubject is the concat of externalId & sourceId, this change will not delete the access token